prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -1132,20 +1132,15 @@ subroutine initProgramVariables(input, env)
     ! Basic variables
     tSccCalc = input%ctrl%tScc
     tDFTBU = input%ctrl%tDFTBU
-    tSpin = input%ctrl%tSpin
```

Comment:
Why do you remove this line? Then, `tSpin` always set to false in the code even though we use spin polarisation in the input file. I think that the line `tSpin = input%ctrl%tSpin` should remain here.

Reply:
Yes, you are correct. Thank you for pointing out the mistake.